packages/fxa-settings/src/components/ConnectedServices/Service.tsx

```diff
@@ -35,6 +38,10 @@ export function Service({
   let serviceLink, Icon;
 
   switch (name) {
+    case 'Add-ons':
```

Comment:
Can we add a couple tests for these cases to ConnectedServices/index.test.tsx or file a follow up item? These extra cases have dropped the statements test coverage from 96.15% to 71.43%, which puts this file's coverage stats in the 'yellow'. Coverage stats can be see by running, `yarn workspace fxa-settings test:coverage`.

Reply:
I can add some.  When I run that command I get `You're likely seeing this bug because you haven't defined a root rescript or your root rescript contains a syntactical error. If you're certain of otherwise, please file an issue.`  Any recommendations for that?